server/aws-lsp-codewhisperer/src/language-server/netTransform/tests/validation.test.ts

```diff
@@ -63,7 +63,7 @@ describe('Test validation functionality', () => {
         expect(validateProject(mockStartTransformationRequest, mockedLogging)).to.equal(true)
     })
 
-    it('should return false when project is not a supported type', () => {
+    it('should return true when project is not a previously supported type', () => {
```

Comment:
This test is a little confusing. Do we consider removing it if we have removed the validation so the test has no meaning?